OOInteraction/src/commands/CAddCalleesToView.cpp

```diff
@@ -53,29 +53,30 @@ Interaction::CommandResult* CAddCalleesToView::executeWithArguments(Visualizatio
 	auto ancestor = source->findAncestorWithNode();
 	auto name = arguments.at(0);
 
-	if (name == "current")
-		name = ancestor->scene()->currentViewItem()->name();
-	auto view = ancestor->scene()->viewItem(name);
+	auto view = ancestor->scene()->currentViewItem();
 
 	if (view)
 	{
 		auto callees_ = callees(ancestor->node());
-		auto pos = view->positionOfNode(ancestor->node());
+		auto pos = view->positionOfItem(ancestor->parent());
 
 		if (callees_.size() > 0)
 		{
+			Model::Node* actualNode;
```

Comment:
I know you initialize this later, but I'd still prefer if you add a  {} here to make it null to begin with.